### PR TITLE
The PHP-Wechaty

### PR DESCRIPTION
**Pull request by shraddhavp**
this is a PR that contains the updated document of the PHP-Wechaty  and links to #890

### CODE REVIEW

File: docusaurus/docs/polyglot/php/overview.md [hunk 1]
```diff
@@ -6,28 +6,39 @@ sidebar_label: PHP
 
 [![PHP Wechaty](https://img.shields.io/badge/Wechaty-PHP-7de)](https://github.com/wechaty/php-wechaty)
 
-- GitHub Repo <https://github.com/wechaty/php-wechaty>
-- Getting Started Template Repo <https://github.com/wechaty/php-wechaty-getting-started>
+PHP-Wechaty is a PHP programming client derived from the Wechaty ecosystem.PHP is a server scripting language and a powerful tool for making dynamic and interactive Web pages. You can find more information on PHP-Wechaty [here](https://github.com/wechaty/php-wechaty). The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates how the PHP-Wechaty can be implemented on the already existing TypeScript Wechaty ecosystem.
```
Comment (Soumi7) [inline docusaurus/docs/polyglot/php/overview.md:9]: ecosystem. <space> PHP

**Review by Soumi7 — APPROVED** [review]
LGTM

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/polyglot/php/overview.md [hunk 2]
```diff
@@ -6,28 +6,39 @@ sidebar_label: PHP
...
+})->onLogin(function(\IO\Github\Wechaty\User\ContactSelf $user) {
+})->onMessage(function(\IO\Github\Wechaty\User\Message $message) {
+    $message->say("hello from PHP7.4");
+})->start();
+```
 
 ## Getting Started
 
-Template Repo: <https://github.com/wechaty/php-wechaty-getting-started>
+Run the below commands for starting the PHP-Wechaty bot. For further information also you can visit [Template repo](https://github.com/wechaty/php-wechaty-getting-started)
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/php/overview.md]:
```suggestion
Run the below commands for starting the PHP-Wechaty bot. For further information, visit [Template repo](https://github.com/wechaty/php-wechaty-getting-started).
```

File: docusaurus/docs/polyglot/php/overview.md [hunk 3]
```diff
@@ -6,28 +6,39 @@ sidebar_label: PHP
 
 [![PHP Wechaty](https://img.shields.io/badge/Wechaty-PHP-7de)](https://github.com/wechaty/php-wechaty)
 
-- GitHub Repo <https://github.com/wechaty/php-wechaty>
-- Getting Started Template Repo <https://github.com/wechaty/php-wechaty-getting-started>
+PHP-Wechaty is a PHP programming client derived from the Wechaty ecosystem.PHP is a server scripting language and a powerful tool for making dynamic and interactive Web pages. You can find more information on PHP-Wechaty [here](https://github.com/wechaty/php-wechaty). The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates how the PHP-Wechaty can be implemented on the already existing TypeScript Wechaty ecosystem.
+The World's shortest PHP chatbot is PHP-Wechaty, which can be very easily implemented and used as shown below.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/php/overview.md]:
```suggestion
The World's shortest PHP chatbot is PHP-Wechaty, which can be very easily implemented and used as shown below:
```

**Review by proudofsimin — COMMENTED** [review]
Please fix some minor mistakes. Thank you.

**Comment by shraddhavp** [comment]
@proudofsimin  done the changes as suggested

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Pull request merged**